src/models/cost_functions/MarketBidCost.jl

```diff
@@ -26,9 +26,17 @@ mutable struct MarketBidCost <: OperationalCost
     "shut-down cost"
     shut_down::Float64
     "Variable Cost TimeSeriesKey"
-    incremental_offer_curves::Union{Nothing, IS.TimeSeriesKey, PiecewiseLinearData}
+    incremental_offer_curves::Union{
+        Nothing,
+        IS.TimeSeriesKey,
+        CostCurve{InputOutputCurve{PiecewiseLinearData}},
+    }
     "Variable Cost TimeSeriesKey"
-    decremental_offer_curves::Union{Nothing, IS.TimeSeriesKey, PiecewiseLinearData}
+    decremental_offer_curves::Union{
+        Nothing,
+        IS.TimeSeriesKey,
+        CostCurve{InputOutputCurve{PiecewiseLinearData}},
+    }
     "Bids for the ancillary services"
     ancillary_services::Vector{Service}
```

Comment:
for future reference we need to refactor the ancillary services bids too and support price/quantity bids

Reply:
Done